neuro_san/http_sidecar/http_sidecar.py

```diff
@@ -65,20 +69,24 @@ def __init__(self, port: int, http_port: int,
 
     def __call__(self):
         """
-        Method to be called by a process running tornado HTTP server
+        Method to be called by a thread running tornado HTTP server
         to actually start serving requests.
         """
         self.lock = threading.Lock()
         self.logger = HttpLogger(self.forwarded_request_metadata)
-
         app = self.make_app()
+
+        event_set = self.start_event.wait(timeout=self.TIMEOUT_TO_START_SECONDS)
+        if not event_set:
+            self.logger.error({}, "Timeout (%d sec) waiting for HTTP server to start", self.TIMEOUT_TO_START_SECONDS)
+            return
+
```

Comment:
Wait for "go" signal which will be set by gRPC server and corresponding machinery.

Reply:
You should move this PR comment into the code.
Not obvious otherwise.

Reply:
OK done.